traffic_ops/testing/api/v4/jobs_test.go

```diff
@@ -36,6 +36,18 @@ func TestJobs(t *testing.T) {
 		GetTestJobs(t)
 		GetTestInvalidationJobs(t)
 		JobCollisionWarningTest(t)
+		GetTestJobsByValidData(t)
+		GetTestJobsByInvalidData(t)
+		CreateTestJobsInvalidds(t)
```

Comment:
nit: can you capitalize `ds` (Delivery Service) in the function, makes it easy to read?

Reply:
updated.